custom-vu/src/main/kotlin/jces1209/vu/action/WorkAnIssue.kt

```diff
@@ -91,4 +96,15 @@ class WorkAnIssue(
             }
         }
     }
+    private fun attachScreenShot(issuePage: AbstractIssuePage) {
+        val attachScreenShot = issuePage.addAttachment()
+        attachScreenShot.makeScreenShot()
+        meter.measure(
+            key = ActionType("Attach ScreenShot") { Unit },
+            action = {
+                attachScreenShot.pasteScreenShot()
+                attachScreenShot.waitForTheScreenShotAttached()
+            }
+        )
```

Comment:
You can use the Groovy-style call:
```suggestion
        meter.measure(ActionType("Attach ScreenShot") { }) {
            attachScreenShot.pasteScreenShot()
            attachScreenShot.waitForTheScreenShotAttached()
        }
```

Reply:
changed

Reply:
Still looks like `action = {` on my end